src/components/Link/index.tsx

```diff
@@ -1,9 +1,9 @@
 import { ReactNode } from 'react'
 
-const Link: React.FC<Props> = ({ to, children }: Props): JSX.Element => {
+const Link: React.FC<Props> = ({ to, children, target }: Props): JSX.Element => {
   return (
-    <a className="text-sm no-underline no-border mt-10" href={to}>
-      <button className="bg-cyan-500 hover:bg-cyan-700 transition ease-in-out rounded px-3 py-2">{children}</button>
+    <a className="text-sm no-underline no-border mt-5" href={to} target={target}>
```

Comment:
Would you mind check `no-border` class since there's a border in preview ?
https://project-hub-kmutt-jpxliiocu-projecthub-kmutt.vercel.app/

Reply:
Also, It seems like other classes aren't working as well for some reason 🤔